src/attr/filters.py

```diff
@@ -22,14 +23,21 @@ def include(*what):
     Include *what*.
 
     :param what: What to include.
-    :type what: `list` of `type` or `attrs.Attribute`\\ s
+    :type what: `list` of classes `type`, field names `str` or
+        `attrs.Attribute`\\ s
 
     :rtype: `callable`
+
+    .. versionchanged:: 22.3.0 Accept field name string as input argument
```

Comment:
We're doing CalVer and it's 2023. :)

```suggestion
    .. versionchanged:: 23.1.0 Accept field name string as input argument
```

Reply:
Whoops